scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -503,6 +506,24 @@ function Set-ProvisioningMode {
     }
 }
 
+function Set-AgentEnvironmentVariables {
```

Comment:
I'm not sure we want to add this to the install script. You've added code in this review to set IOTEDGE_HOST in the quickstart code, why not set RuntimeLogLevel there too? Don't we always want to set it to the same value when we run quickstart?

Reply:
Think again and checked that Linux installation doesn't support to pass runtime log level as well.  I will remove it and handle it in IoT Edge Quickstart.